src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code SortCommand}.
+ */
+public class SortCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_sortSuccess() {
+        SortCommand command = new SortCommand();
+        expectedModel.sortCompanyList();
```

Comment:
then for the expectedModel you will not have to sort already since its what you expect and you trying to test the sort command, so i think should not use the sort command